src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/Rules/MembersMustExistTests.cs

```diff
@@ -483,5 +483,44 @@ public class ThirdNested
 
             AssertExtensions.MultiRightEmptyDifferences(expectedLeftMetadata, rightSyntaxes.Length, differences);
         }
+
+
+
+        [Fact]
+        public void ParameterlessConstructorRemovalIsReported()
+        {
+            string leftSyntax = @"
+namespace CompatTests
+{
+  public sealed class First
+  {
+    public First() { }
```

Comment:
nit: can we omit the explicit public constructor declaration?  That way it's more indicative of the user action.  Someone won't realize that they had a public constructor when they add an internal/private one.

Reply:
Sure, we can do that.

Reply:
```suggestion
```